Documentation.app/Contents/MacOS/First steps.playground/Pages/How to use carbon api?.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,59 @@
+// nef:begin:header
+/*
+ layout: docs
+ title: How to use carbon api?
```

Comment:
The file name should be changed accordingly, but I cannot do it from here.

```suggestion
 title: How to use Carbon API?
```